extensions/wikia/SitemapPage/SitemapPageModel.class.php

```diff
@@ -10,16 +10,7 @@ class SitemapPageModel extends WikiaModel {
 	const MIN_LEVEL = 1;
 	const VERTICAL_UNKNOWN = 'Unknown';
 
-	protected static $verticalNames = [
-		WikiFactoryHub::VERTICAL_ID_OTHER => 'Other',
-		WikiFactoryHub::VERTICAL_ID_TV => 'TV',
-		WikiFactoryHub::VERTICAL_ID_VIDEO_GAMES => 'Games',
-		WikiFactoryHub::VERTICAL_ID_BOOKS => 'Books',
-		WikiFactoryHub::VERTICAL_ID_COMICS => 'Comics',
-		WikiFactoryHub::VERTICAL_ID_LIFESTYLE => 'Lifestyle',
-		WikiFactoryHub::VERTICAL_ID_MUSIC => 'Music',
-		WikiFactoryHub::VERTICAL_ID_MOVIES => 'Movies',
-	];
+	protected $verticals = null;
```

Comment:
Any harm in just initializing `$verticals` here and remove the need for the `getVerticals()` function below?